test/functional/config.js

```diff
@@ -65,11 +67,13 @@ testingEnvironments[testingEnvironmentNames.mobileBrowsers] = {
         accessKey: process.env.BROWSER_STACK_ACCESS_KEY
     },
 
+    unstableNetworkMode: true,
+
     browsers: [
         {
             realMobile: true,
             os:         'android',
-            osVersion:  '7.1',
+            osVersion:  '8.0',
```

Comment:
At present, the max avalible version of Android on Saucelabs is 7.1 (https://wiki.saucelabs.com/display/DOCS/Supported+Android+Emulators).
Let's update it after Andorid 8.0 will be avalible on Saucelabs.